integration/testutil_soci.go

```diff
@@ -118,8 +119,8 @@ func buildIndex(sh *shell.Shell, src imageInfo, opt ...indexBuildOption) string
 }
 
 func validateSociIndex(sh *shell.Shell, sociIndex soci.Index, imgManifestDigest string, includedLayers map[string]struct{}) error {
-	if sociIndex.MediaType != ocispec.MediaTypeArtifactManifest {
-		return fmt.Errorf("unexpected index media type; expected = %v, got = %v", ocispec.MediaTypeArtifactManifest, sociIndex.MediaType)
+	if sociIndex.MediaType != ocispec.MediaTypeArtifactManifest && sociIndex.MediaType != ocispec.MediaTypeImageManifest {
+		return fmt.Errorf("unexpected index media type; expected = %v or %v, got = %v", ocispec.MediaTypeArtifactManifest, ocispec.MediaTypeImageManifest, sociIndex.MediaType)
```

Comment:
nit: `expected = %v or %v, got = %v` -> `expected: [%v, %v], got: %v`

Reply:
I disagree with "[foo, bar]" as shorthand for "foo or bar" in contexts like this. The brackets look more like an array of length two.

Reply:
I think the the the `[]` notation is fine if we explicitly mention a plural like **types** (eg: `expected types [a,b]`)